From 00c4b331df8157dff7c251323d00ec2d63691dfb Mon Sep 17 00:00:00 2001
From: Miklos Vajna <vmiklos@collabora.com>
Date: Mon, 28 Jan 2019 09:04:56 +0100
Subject: [PATCH] Add missing include

CMXParser.cpp:42:43: error: expected ')'
    CDR_DEBUG_MSG(("invalid tag length %" PRIu16 "\n", tagLength));
                                          ^
CMXParser.cpp:42:19: note: to match this '('
    CDR_DEBUG_MSG(("invalid tag length %" PRIu16 "\n", tagLength));
                  ^
1 error generated.

Change-Id: I9575a96327a62dc6c44950d35a8740242e1a4433
---
 src/lib/CMXParser.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib/CMXParser.cpp b/src/lib/CMXParser.cpp
index 78cc417..02852c9 100644
--- a/src/lib/CMXParser.cpp
+++ b/src/lib/CMXParser.cpp
@@ -14,6 +14,7 @@
 #include <math.h>
 #include <stdlib.h>
 #include <utility>
+#include <cinttypes>
 
 #include "libcdr_utils.h"
 #include "CDRPath.h"
-- 
2.21.0

